reddit_liveupdate/public/static/js/liveupdate/statusBar.js

```diff
@@ -30,6 +30,20 @@
     },
   })
 
+  function formatTotalViews(totalViews) {
+    var totalViewsString = r.P_('%(num)s total view', '%(num)s total views', totalViews)
+
+    if (totalViews < 999) {
+      var formattedTotalViews = totalViews
+    } else if (totalViews <= 999999) {
+      var formattedTotalViews = (totalViews / 1000).toFixed(1) + 'k'
+    } else {
+      var formattedTotalViews = (totalViews / 1000000).toFixed(2) + 'm'
```

Comment:
What about billions!?!?

Reply:
that'd be pretty neat!

(just to be clear: this is a JS port of `r2.lib.strings.formatted_view_count`)